### PR TITLE
Support `KHR_materials_variants`

### PR DESCRIPTION
**Pull request by expenses**

### CODE REVIEW

File: src/mesh/mod.rs [hunk 1]
```diff
@@ -259,6 +259,30 @@ impl<'a> Primitive<'a> {
...
+                    document: self.mesh.document,
+                    iter: (&[]).iter(),
+                }
+            }
+        } else {
+            iter::Mappings {
+                document: self.mesh.document,
+                iter: (&[]).iter(),
+            }
+        }
```
Comment (IcanDivideBy0) [inline src/mesh/mod.rs:283]: same:
```rust
let iter = self
    .json
    .extensions
    .as_ref()
    .and_then(|extensions| extensions.khr_materials_variants.as_ref())
    .map(|variants| variants.mappings.iter())
    .unwrap_or_else(|| (&[]).iter());

iter::Mappings {
    document: self.mesh.document,
    iter,
}
```

**Review by IcanDivideBy0 — APPROVED** [review]
Comments are not blocking, can me merged as is

**Comment by expenses** [comment]
@IcanDivideBy0 all good suggestions, thanks!

**Comment by IcanDivideBy0** [comment]
Thank you!
We're planning a release including your changes in the upcoming days :)

**Pull request merged**